stbtrace/nfs.st

```diff
@@ -0,0 +1,336 @@
+#
+# Copyright (c) 2019 by Delphix. All rights reserved.
+#
+
+from bcc import BPF
+from time import sleep
+from time import time
+import datetime
+import ctypes as ct
+import sys
+sys.path.append('/opt/delphix/server/bin')
+from bcchelper import BCCHelper
+
+# BPF txg program
+bpf_text = """
+#include <uapi/linux/ptrace.h>
+#include <linux/bpf_common.h>
+#include <uapi/linux/bpf.h>
+#include <linux/sunrpc/svc.h>
+#include "/opt/delphix/server/etc/bcc_helper.h"
```

Comment:
I think it would make sense to ship this file as part of this package (and include it in this PR) because the next step will be to remove all of the consumers of this include from dlpx-app-gate, reducing its consumers to only scripts in this repo.

Reply:
I added the lib directory and the bcc files.  I wasn't sure where those would be installed so I didn't change the include statements.  I did modify the python code to find bcchelper, stealing that code from estat.